blockchain.go

```diff
@@ -612,11 +619,6 @@ func (b *Blockchain) newFVMContextFromHeader(header *flowgo.Header) fvm.Context
 	return fvm.NewContextFromParent(
 		b.vmCtx,
 		fvm.WithBlockHeader(header),
-		fvm.WithReusableCadenceRuntimePool(
-			reusableRuntime.NewReusableCadenceRuntimePool(
-				1,
-				runtime.Config{Debugger: b.debugger, AccountLinkingEnabled: true}),
-		),
```

Comment:
is there a reason to remove this? this should be needed for debugger to function ( not sure though, flow-go is changing too fast )

Reply:
I had to remove this line here, because it would create a new `CoverateReport` object, which would be ephemeral during script & transaction execution.

The `ReusableCadenceRuntimePool` is properly initialized here: https://github.com/onflow/flow-emulator/blob/master/blockchain.go#L427-L446, so with:

```go
return fvm.NewContextFromParent(
    b.vmCtx,
    fvm.WithBlockHeader(header),
)
```

the `b.vmCtx` contains all the necessary config for the `Debugger`, `AccountLinkingEnabled` and `CoverageReportingEnabled` from the initial setup.

Reply:
ah you have right, I missed that